frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -255,24 +254,21 @@ const InfoControl: React.FC<InfoControlProps> = ({ open, setOpen, onHeaderAction
             <InfoIcon />
           </InfoButton>
         </TooltipWrapper>
-        {open &&
-          popUpContext.propertyInfo &&
-          canViewProperty &&
-          popUpContext.propertyTypeID === PropertyTypes.PARCEL && (
-            <TooltipWrapper toolTipId="associated-buildings-id" toolTip="Associated Buildings">
-              <BuildingsButton
-                id="slideOutBuildings"
-                variant="outline-secondary"
-                className={clsx({ open })}
-                onClick={() => {
-                  setAsscBuildingsOpen(true);
-                  setGeneralInfoOpen(false);
-                }}
-              >
-                <BuildingSvg className="svg" />
-              </BuildingsButton>
-            </TooltipWrapper>
-          )}
+        {open && popUpContext.propertyInfo && canViewProperty && isParcel && (
```

Comment:
I have some changes here that will go through first I think, so this logic will change a bit